portal/src/views/StationsView.vue

```diff
@@ -1,31 +1,53 @@
 <template>
     <StandardLayout @sidebar-toggle="onSidebarToggle" :viewingStations="true">
-        <div class="container-map">
-            <StationsMap @show-summary="showSummary" :mapped="mapped" :layoutChanges="layoutChanges" v-if="mapped" />
-            <StationSummary
-                v-if="activeStation"
-                class="summary-container"
-                @close="closeSummary"
-                :station="activeStation"
-                v-bind:key="activeStation.id"
-            />
-            <div v-if="isAuthenticated && showNoStationsMessage && hasNoStations" id="no-stations">
-                <div id="close-notice-btn" v-on:click="closeNotice">
-                    <img alt="Close" src="@/assets/close.png" />
-                </div>
-                <p class="heading">Add a New Station</p>
-                <p class="text">
-                    You currently don't have any stations on your account. Download the FieldKit app and connect your station to add them to
-                    your account.
-                </p>
-                <a href="https://apps.apple.com/us/app/fieldkit-org/id1463631293?ls=1" target="_blank">
-                    <img alt="App store" src="@/assets/appstore.png" class="app-btn" />
-                </a>
-                <a href="https://play.google.com/store/apps/details?id=com.fieldkit&hl=en_US" target="_blank">
-                    <img alt="Google Play" src="@/assets/googleplay.png" class="app-btn" />
-                </a>
+        <div class="view-type-container">
```

Comment:
I think eventually we'll move these to separate components, rather than inline templates.